static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -54,6 +54,41 @@
     position: relative;
 }
 
+.youtube-media-atom__paid-for-overlay {
+    background-color: #ffffff;
```

Comment:
We try to avoid the use of hex codes within the CSS, prefering to use sass variables that map to the colour codes,  these are defined in `static/src/stylesheets/_vars.scss`, fir example `$brightness-100` maps  to `#ffffff`.